src/utilities/network/lookupApi.js

```diff
@@ -73,6 +102,7 @@ export const createPatientRecord = response => {
     isVisible: true,
   };
   UIDatabase.write(() => UIDatabase.update('Name', patient));
+  policies.forEach(policy => createInsurancePolicyRecord(policy));
```

Comment:
Just for fun and to remove a tiny bit of code:

```suggestion
  policies.forEach(createInsurancePolicyRecord);
```

Reply:
Fixed in #2671 (fcdd2832).